README.md

```diff
@@ -76,48 +76,45 @@ python -m pip install molecule 'molecule-plugins[docker]' docker
 
 ### Testing the roles using Molecule
 
-Molecue 6.0 requires that the test configuration is not in the top-level
-directory of the collection. To support running the tests with Molecule 6, the
-Molecule configuration is in `ansible_collections/mirsg/infrastructure/tests`.
-To run the tests you must be in this directory:
+Each role has its own Molecule configuration, which can be found it the `molecule/`
+folder of each role. Molecule base configurations are used to reduce to amount
+of duplication in the setup for testing each role. There are two base configurations
+that correspond to two [Molecule
+scenarios](https://ansible.readthedocs.io/projects/molecule/getting-started/#molecule-scenarios)
+one for testing on CentOS 7 and another for testing on Rocky 9. The base configurations
+are in the `ansible_collections/mirsg/infrastructure/tests` folder.
+
+To run the tests for a specific role, first navigate the the role directory, e.g.
 
 ```shell
-cd ansible_collections/mirsg/infrastructure/tests
+cd ansible_collections/mirsg/infrastructure/roles/provision
 ```
 
-This collection contains [molecule
-scenarios](https://ansible.readthedocs.io/projects/molecule/getting-started/#molecule-scenarios)
-that allow testing of the individual roles. There are scenarios that run the
-tests on both CentOS 7 and Rocky 9.
-
-To run the CentOS 7 tests for the roles:
+Then invoke Molecule with a base config and corresponding scenario. To run the
+tests on CentOS 7:
 
 ```shell
-molecule test -s centos7_roles
+molecule --base-config ../../tests/centos7_base_config.yml test -s centos7
```

Comment:
Since we're using the long form of the `--base-config` flag, should we also use the long form for `--scenario` (instead `-s`)?

Reply:
yep that would be clearer